dbt_sugar/core/connectors/base.py

```diff
@@ -30,10 +30,8 @@ def __init__(
         self.engine = sqlalchemy.create_engine(**connection_params)
 
     def get_columns_from_table(
-        self,
-        target_table: str,
-        target_schema: str,
-    ) -> Optional[List[Tuple[Any]]]:
+        self, target_table: str, target_schema: str, **kwargs
```

Comment:
what is this argument **kwargs
is missing in the documentation

Reply:
It's for the snowflake connector which may need to get the argument for `use_describe`. How would you document it?